vm/thread.go

```diff
@@ -204,12 +204,12 @@ func (t *thread) evalMethodObject(receiver Object, method *MethodObject, receive
 	argTypesCount := len(method.argTypes())
 
 	for _, at := range method.argTypes() {
-		if at == bytecode.NormalArg {
+		if at == bytecode.NormalArg || at == bytecode.RequiredKeywordArg {
```

Comment:
I think we should totally separate the keyword argument's check logic and other argument's check logic. Include required keyword argument into the minimum argument number count is not very intuitional for what/how we're going to check later.